src/sidebar/components/Annotation/AnnotationShareControl.js

```diff
@@ -60,6 +60,9 @@ function AnnotationShareControl({
   const closePanel = () => setOpen(false);
 
   // Interactions outside of the component when it is open should close it
+  //
+  // @ts-expect-error - `useElementShouldClose` requires a non-nullable ref,
+  // but `shareRef` is nullable.
   useElementShouldClose(shareRef, isOpen, closePanel);
```

Comment:
There is a bug here, but since it is not a new one, I just suppressed the error for the moment.

Reply:
`shareRef` is a non null ref, why not to use the pattern you suggested: `const shareRef = /** @type {{ current: HTMLDivElement }} */ (useRef());`

Reply:
`shareRef` is a nullable ref! - There is a `return null` statement in the component mid-way down with an issue link above it.

Reply:
Yes, you are right. I was looking for that closer to the return and I didn't find it...

Then, yes I agree that adding the `ts-expect-error` is the best option for now.